### PR TITLE
SME, QE, and doc peer review pull request for BXMSDOC-3088 Executing a business process Business Central for 7.1

### PR DESCRIPTION
**Pull request by michelehaglund**
Rendered output: http://file.rdu.redhat.com/~mhaglund/BXMSDOC-3088/
Original Jira: https://issues.jboss.org/browse/BXMSDOC-3088

### CODE REVIEW

File: doc-content/enterprise-only/processes/generate-forms-proc.adoc [hunk 1]
```diff
@@ -10,7 +10,7 @@ You have created the pizza orders business process. For more information, see <<
 .Procedure
 
 . Log in to {CENTRAL} and click *Menu* -> *Design* -> *Projects* -> *pizzaPlace* -> *pizzaOrders*.
-. Click the following menu option and select *Generate all forms*.
+. Select *Generate all forms* from the *Form Generation* pull-down menu.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/processes/generate-forms-proc.adoc]: Consider deleting pull-down. If you do, search the doc for other instances.
IBMSG: pull down adj
Do not use. Use “pull-down,” but only if you must specify the type of menu or list.

File: doc-content/enterprise-only/processes/creating-new-users-proc.adoc [hunk 2]
```diff
@@ -4,17 +4,32 @@ You can create as many {CENTRAL} users as you require. User privileges and setti
...
-. Click the *Roles* tab, click *Add roles*, and select *admin* and *user*.
-. Click *Add to selected roles* and click *Create* to assign `Katy` to the *admin* and *user* roles.
+. Click image:Designer/new-user.png[], enter `Katy`, and click *Next*.
+. Click the *Groups* tab, click image:Designer/add-groups.png[].
+. Select *rest-all* and *kie-server* in the *Group selection for Katy* window and click *Add to selected groups*.
++
+.Group selection for *Katy*
+image::Designer/select-group.png[Selecting group assignments for Katy]
+
+. Click the *Roles* tab, click *Add roles*.
```
[the remark this answers is not in the document]
Comment (michelehaglund) [reply doc-content/enterprise-only/processes/creating-new-users-proc.adoc]: Done. Good catch.

File: doc-content/enterprise-only/processes/creating-new-users-proc.adoc [hunk 3]
```diff
@@ -4,17 +4,32 @@ You can create as many {CENTRAL} users as you require. User privileges and setti
...
+.Add roles for *Katy*
+image::Designer/add-roles.png[Add roles menu location]
+
+. Select *admin* and *user* and click *Add to selected roles*.
+. Click *Create* to assign `Katy` to the *admin* and *user* roles.
 . Click *Yes* to set the password to `katy` and click *Change*.
+. Click the *Users* tab and click image:Designer/new-user.png[].
 . Click *New user*, enter `Bill`, and click *Next*.
-. Click the *Groups* tab, click *Add to groups*.
+. Click the *Groups* tab, click image:Designer/add-groups.png[].
```
[the remark this answers is not in the document]
Comment (michelehaglund) [reply doc-content/enterprise-only/processes/creating-new-users-proc.adoc]: Done.

File: doc-content/enterprise-only/processes/generate-forms-proc.adoc (hunk 1 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/processes/generate-forms-proc.adoc]: Done throughout. Thanks!

File: doc-content/enterprise-only/processes/pizza-confirm-task-proc.adoc [hunk 4]
```diff
@@ -3,11 +3,12 @@
 Create an order confirmation user task, which will return the order details to the customer for confirmation before the final order is placed.
 
 .Procedure
-. Click the Process Order task to create an outgoing connection to a new user task.
+. Click the Process Order task to create an outgoing connection to a new task.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/processes/pizza-confirm-task-proc.adoc]: In that case, make it bold ;-)

**Pull request merged**